internal/slack/users_from_conversation.go

```diff
@@ -50,16 +46,16 @@ func (s *client) getAllUsersFromConversation(chatId string) ([]data.User, error)
 			ChannelID: chatId,
 			Cursor:    cursor,
 		}
-		userIDs, nextCursor, err := s.superBotClient.GetUsersInConversation(params)
+		userIDs, nextCursor, err := s.botClient.GetUsersInConversation(params)
 		if err != nil {
-			return nil, err
+			return nil, errors.Wrap(err, "failed to get users in conversation")
 		}
 
 		// Getting information about each user
 		for _, userID := range userIDs {
-			user, err := s.superBotClient.GetUserInfo(userID)
+			user, err := s.botClient.GetUserInfo(userID)
```

Comment:
you do a lot of requests in this method, so Pqueue is useless here. At least 2 (1 - GetUsersInConversation and 2 - GetUserInfo) requests, but we can have a lot of users in one channel, so it can be 11, 21, 41 ... request. It's better to split somehow, because each iteration (while nextCursor is not empty) can perform a lot of requests, but we handle it as one by passing `GetConversationUsers` to pqueue.

May be to create some wrapper that will handle it with queues or just pass queue here (but there is can be some troubles with helpers package). These are just a few solutions that came up immediately to my mind, so may be you would come up with something better. Moreover, this approach could be used for pagination refactor. So again some thing to think about.